xcube/webapi/ows/stac/controllers.py

```diff
@@ -482,47 +495,76 @@ def _get_single_dataset_collection(
             {
                 'rel': 'items',
                 'href': f'{base_url}{PATH_PREFIX}/collections/'
-                        f'{dataset_id}/items',
+                f'{dataset_id}/items',
```

Comment:
I preferred the previous format (same applies for entries below).

Reply:
So do I. Sorry about all the pure formatting changes in this file. I would never deliberately add them to such a large PR, but PyCharm "helpfully" did this for me while editing and by the time I noticed, its "improvements" were so mixed up with my own code changes that I could only undo them by hand -- and I missed quite a few, such as that one. I've now fixed all (I think) the "weird string indent" reformattings as well as a few others, which should make things marginally less painful for the next reviewers.